frontend/src/scenes/error-tracking/errorTrackingActivityDescriber.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+    ActivityChange,
+    ActivityLogItem,
+    ChangeMapping,
+    defaultDescriber,
+    Description,
+    HumanizedChange,
+    userNameForLogItem,
+} from 'lib/components/ActivityLog/humanizeActivity'
+import { SentenceList } from 'lib/components/ActivityLog/SentenceList'
+
+import { ErrorTrackingGroup } from '~/queries/schema'
+import { ActivityScope } from '~/types'
+
+const errorTrackingGroupActionsMapping: Record<
+    keyof ErrorTrackingGroup,
+    (change?: ActivityChange, logItem?: ActivityLogItem) => ChangeMapping | null
+> = {
+    assignee: () => null,
+    merged_fingerprints(change: ActivityChange | undefined): ChangeMapping | null {
+        const lenBefore = Array.isArray(change?.before) ? change?.before.length || 0 : 0
+        const lenAfter = Array.isArray(change?.after) ? change?.after.length || 0 : 0
+        const mergeCountChange = lenAfter - lenBefore
+        return {
+            // TODO need to be able to name the group here?
+            description: [<>merged {mergeCountChange} groups into this one.</>],
+        }
+    },
+    status: () => null,
```

Comment:
i'm thinking these are the only changes that are meaningful to describe?

Reply:
👍

Reply:
Probably "source map uploaded" but that's not relevant yet